src/cosalib/ova.py

```diff
@@ -63,14 +63,15 @@ def __init__(self, **kwargs):
         # Ensure that coreos.ovf is included in the tar
         self.ovf_path = os.path.join(self._tmpdir, "coreos.ovf")
 
-    def generate_ovf_parameters(self, vmdk, cpu=2,
-                                memory=4096, system_type="vmx-13",
-                                os_type="rhel7_64Guest", scsi="VirtualSCSI",
-                                network="VmxNet3"):
+    def generate_ovf_parameters(self, vmdk, cpu=2, memory=4096):
         """
         Returns a dictionary with the parameters needed to create an OVF file
-        based on the qemu, vmdk, and info from the build metadata
+        based on the qemu, vmdk, image.yaml, and info from the build metadata
         """
+        image_yaml = flatten_image_yaml('src/config/image.yaml')
+
+        system_type = 'vmx-{}'.format(image_yaml.get('vmware-hw-version', 13))
```

Comment:
Instead of having the default in Python code, let's add it to `image-default.yaml` which is intended to act also as the canonical documentation for what can appear in that file.